src/main/client/src/components/@commons/StampComponent.tsx

```diff
@@ -0,0 +1,28 @@
+import { Box, Center, Flex, Icon, Stat, StatHelpText, StatLabel } from '@chakra-ui/react'
+import React from 'react'
+import { FaStamp } from 'react-icons/fa'
+
+interface StampComponentProps {
+  title?: string
+  type: string
+}
+
+export const StampComponent: React.FC<StampComponentProps> = ({ title, type }: StampComponentProps) => {
```

Comment:
I would make it gray as the CardListItem at the community list. But just a suggestion, will talk about it later.